### ISSUE TITLE
Measure and Limit Storage

### ISSUE THREAD

**Issue by vLooz** [lavab/web#894]
Are these mocks enough? @piggyslasher 

![bildschirmfoto 2015-06-29 um 17 16 21](https://cloud.githubusercontent.com/assets/11177812/8411160/c2b57a30-1e82-11e5-9fbc-f2fc82bd25ab.png)
![bildschirmfoto 2015-06-29 um 17 16 06](https://cloud.githubusercontent.com/assets/11177812/8411161/c41ea90a-1e82-11e5-9b55-ad7b232b375d.png)

**Comment by piggyslasher** [comment]
yes actually I've already styled the progress bar ages ago. then it was removed. let me double check with these new guidelines.

**Comment by vLooz** [comment]
alright, cool :-) I just updated them slightly. 
Now they fit the latest settings mocks, you worked with a while ago.

**Comment by piggyslasher** [comment]
I'll style it today.

On Mon, 29 Jun 2015 at 23:41 Felix von Looz <notifications@github.com>
wrote:

> alright, cool :-) I just updated them slightly.
> Now they fit the latest settings mocks, you worked with a while ago.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lavab/web/issues/894#issuecomment-116736827>.
>
-- 
Vik Ramanujam
vik@ramanuj.am (official)
piggyslasher@gmail.com (everything else)
+6016 656 4647

**Comment by joergG** [comment]
Brief Functional Description:
- we will NOT delete old mails to limit storage (like slack)
- measure storage and show the user an actual storage-amount and/or storage remaining (see mocks)
- when limit (500MB for basic accounts) is reached, no more emails to be accepted and inform the specific sender that the mailbox is full (Autoreply)
- also when mails cannot be accepted anymore deliver a "warning email" to Inbox
- Supporters have individual limits according to their packages
- Closed Beta Users have individual limits (1GB)

**Comment by piggyslasher** [comment]
again, I've styled this as per the current settings styling. It would look more in tune with fvls new design when I finish #737

**Comment by piggyslasher** [comment]
@let4be - all yours

**Comment by joergG** [comment]
@andreis - afaik it is your turn first then let4be?

**Comment by joergG** [comment]
@ALL - waffle will work different from trello: we'll leave this issue in Implementation and just assign the next person to do sth. until the feature is finished. that's why I asked andrei above...

**Comment by joergG** [comment]
@let4be - can you do sth. here or do you need BE first?

**Comment by let4be** [comment]
Going to add some basic stuff here, like dialogs for styling

**Comment by let4be** [comment]
this needs back-end first(previous comment meant to be for another ticket)

**Comment by joergG** [comment]
Here we estimated:
- 5h BE
- 2h FE
- 1h UI

@piggyslasher @andreis @let4be - please re-estimate if needed

**Comment by piggyslasher** [comment]
There's no work left for me on this. I completed this in last sprint.

On Thu, 16 Jul 2015 at 20:36 joergG <notifications@github.com> wrote:

> Here we estimated:
>
>    - 5h BE
>    - 2h FE
>    - 1h UI
>
> @piggyslasher <https://github.com/piggyslasher> @andreis
> <https://github.com/andreis> @let4be <https://github.com/let4be> - please
> re-estimate if needed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lavab/web/issues/894#issuecomment-121945095>.
>
-- 
Vik Ramanujam
vik@ramanuj.am (official)
piggyslasher@gmail.com (everything else)
+6016 656 4647
